flytekit/core/container_task.py

```diff
@@ -10,6 +10,7 @@
 from flytekit.models.security import Secret, SecurityContext
 
 
+# TODO: do we need pod_template here? Seems that it is a raw container not running in pods
```

Comment:
Yes, the goal is to add this support to `ContainerTask` as well. So these are defined by containers, but Flyte executes them as a `Pod`. So if a user provides a `PodTemplate` argument, we should populate the oneof in idl as a `K8sPod` rather than a `Contianer`.

Reply:
1. I didn't see any reference of `ContainerTask` in flytekit source code. Was it deprecated in the core and only used by developers in the plugin?
2. Why the `get_command` part only print instead of return `list[str]`? And why the printing message is`docker run...`?